CONTRIBUTING.md

```
@@ -52,14 +53,28 @@ installation.
+### Optional Requirements
+
+- [cargo-deb](https://github.com/kornelski/cargo-deb#installation) for debian bundle packaging.
+- [cargo-edit](https://github.com/killercup/cargo-edit). for `cargo upgrade`-related commands (i.e.: `make upgrade`)
+- [cargo-watch](https://github.com/watchexec/cargo-watch#install)
+  for `cargo watch`-related commands (i.e.: `make test-*-watch`).
+- [ggshield](https://github.com/GitGuardian/ggshield#installation) for `yarn test:sec` command.
+
+#### Debian-based OS
+
+- [nsis](https://nsis.sourceforge.io/Main_Page):
+  ```sh
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)